scripts/generate-runner-parameters-documentation.py

```diff
@@ -66,19 +66,21 @@ def main():
         result.append(HEADER)
         result.append('')
 
-        for name, values in runner['runner_parameters'].items():
-            format_values = {'name': name}
-            format_values.update(values)
-            line = '* ``%(name)s`` (%(type)s) - %(description)s' % format_values
-            result.append(line)
-
-        file_name = runner['name'].replace('-', '_')
-        path = '../docs/source/_includes/runner_parameters/%s.rst' % (file_name)
-        destination_path = os.path.join(CURRENT_DIR, path)
-        result = '\n'.join(result)
-
-        with open(destination_path, 'w') as fp:
-            fp.write(result)
+        runner_parameters = runner.get('runner_parameters', None)
```

Comment:
LGTM, although I would prefer `runner.get('runner_parameters', {})`, then we can get rid of `if` and one level of indentation :P

Reply:
Won't work in this case since the file generation and writing etc is needed only when runner_parameters are present but otherwise your suggestion is valid.